gatsby/src/docs/sdks/javascript/index.mdx

```diff
@@ -108,11 +110,15 @@ Learn more about what releases can do, including using a repository integration,
 
 Sentry supports un-minifying JavaScript via source maps. This lets you view source code context obtained from stack traces in their original untransformed form, which is particularly useful for debugging minified code (for example, UglifyJS), or transpiled code from a higher-level language (for example, TypeScript, ES6). 
 
-We **highly recommend** you incorporate source maps with the Sentry JavaScript SDK to receive the full benefit of error tracking and monitoring. Source maps provide stack traces, which will provide more information regarding errors. 
+We **highly recommend** you incorporate source maps to receive the full benefit of error tracking and monitoring.
 
-When you’re using the Sentry JavaScript SDK, the SDK automatically fetches the source code and source maps by scraping the URLs within the stack trace. However, you may disable the JavaScript source fetching by logging in to Sentry and toggling off `Allow JavaScript source fetching` in your organization's settings for Security & Privacy. See our [source maps documentation](/platforms/javascript/sourcemaps) to learn more.
+When you’re using the Sentry JavaScript SDK, the SDK automatically fetches the source code and source maps by scraping the URLs within the stack trace. If you need to disable the JavaScript source fetching, log in to Sentry and toggle off `Allow JavaScript source fetching` in your organization's settings for Security & Privacy. See our [source maps documentation](/platforms/javascript/sourcemaps) to learn more.
 
-Upload source maps using our [Webpack plugin](/platforms/javascript/sourcemaps/#webpack), [Uglify JS](/platforms/javascript/sourcemaps/#uglifyjs), [System JS](/platforms/javascript/sourcemaps/#systemjs), or [TypeScript](/platforms/javascript/sourcemaps/#typescript).
+Upload source maps using: 
```

Comment:
This is a partially incorrect statement. Source Maps can be uploaded only using `Webpack plugin` or `sentry-cli` itself. UglifyJS, SytemJS, and TypeScript can only generate them. Not sure how to reword it though.

Reply:
Oh. Good catch. Thanks. Modified to:
"Upload source maps using either our [Webpack plugin](/platforms/javascript/sourcemaps/#webpack) or the [Sentry CLI](cli/). Sourcemaps can be generated by [Uglify JS](/platforms/javascript/sourcemaps/#uglifyjs), [System JS](/platforms/javascript/sourcemaps/#systemjs), or [TypeScript](/platforms/javascript/sourcemaps/#typescript)."